src/components/SearchIcon.tsx

```diff
@@ -0,0 +1,12 @@
+import { ReactElement } from 'react';
+
+interface SearchIconProps {
+	onClick: () => void
+}
+
+export function SearchIcon({ onClick }: SearchIconProps): ReactElement {
```

Comment:
This is definitely not a best way to insert image. Image should be in separate `.svg` and be wrapped by button. It's important for accessibility